x-pack/plugins/maps/server/saved_objects/setup_saved_objects.ts

```diff
@@ -61,23 +61,6 @@ export function setupSavedObjects(
       );
     },
   });
-
```

Comment:
Thanks, this resolves https://github.com/elastic/kibana/issues/139347 for maps.

Question, will this impact migrations? If a cluster has enabled, will the cluster be able to upgrade. We have seen some SDHs regarding migration failures for maps-telemetry and enabled.

Reply:
By adding it to the `REMOVED_TYPES` list in `packages/core/saved-objects/core-saved-objects-migration-server-internal/src/core/unused_types.ts` any remaining maps-telemetry documents will be deleted upon the next upgrade. So it won't impact upgrades.